.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+permissions:
+  contents: read
+jobs:
+  go:
+    name: Lint (Go)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.17.8" # The Go version to download (if necessary) and use.
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.45.2
+          args: --timeout=5m
```

Comment:
Can we add a [protolint](https://github.com/marketplace/actions/protolint) job here too? And maybe a `dev/lint` script that calls `protolint` with `-fix`?

Reply:
Done